test/functional/legacy/076_completefunc_spec.lua

```diff
@@ -0,0 +1,68 @@
+-- Tests for completefunc/omnifunc.
+
+local helpers = require('test.functional.helpers')
+local feed, insert, source = helpers.feed, helpers.insert, helpers.source
+local clear, expect, execute = helpers.clear, helpers.expect, helpers.execute
+
+describe('completefunc', function()
+  setup(clear)
+
+  it('is working', function()
+    insert([=[
+      +++
+      one
+      two
+      three]=])
+
+    -- Test that nothing happens if the 'completefunc' opens.
+    -- A new window (no completion, no crash).
```

Comment:
The conversion script likes to change each line into a sentence, but it's not correct here. I'll fix before push.